src/Features/CSharp/Portable/Structure/Providers/StringLiteralExpressoinStructureProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Structure;
+
+namespace Microsoft.CodeAnalysis.CSharp.Structure
+{
+    internal sealed class StringLiteralExpressoinStructureProvider : AbstractSyntaxNodeStructureProvider<LiteralExpressionSyntax>
```

Comment:
I think you need to add this new provider to the list in [`CSharpBlockStructureProvider.cs`](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/Structure/CSharpBlockStructureProvider.cs)